stable/pipelines/templates/metadata/grpc-configmap.yaml

```diff
@@ -4,9 +4,9 @@ kind: ConfigMap
 metadata:
   name: metadata-grpc-configmap
   labels:
-    component: metadata-server
+    component: metadata-grpc-server
 {{ include "pipelines.commonLabels" . | indent 4 }}
 data:
-  METADATA_GRPC_SERVICE_HOST: "metadata-service"
+  METADATA_GRPC_SERVICE_HOST: "metadata-grpc-server"
```

Comment:
The service name is now `metadata-grpc-service` this looks like it needs to be the same as https://github.com/v3io/helm-charts/pull/488/files#diff-d9fc3ec5e5971729df4efe8261f34cc8R5

Reply:
You are correct, it should be service. I'm checking to see everything works and then I'll update the PR